checks/folder_check.go

```diff
@@ -9,17 +9,21 @@ import (
 )
 
 type Filesystem interface {
-	Close()
+	Close() error
 	ReadDir(name string) ([]os.FileInfo, error)
 	Stat(name string) (os.FileInfo, error)
 }
 
 type FolderCheck struct {
-	Oldest  os.FileInfo
-	Newest  os.FileInfo
-	MinSize os.FileInfo
-	MaxSize os.FileInfo
-	Files   []os.FileInfo
+	Oldest                os.FileInfo
+	Newest                os.FileInfo
+	MinSize               os.FileInfo
+	MaxSize               os.FileInfo
+	SupportsTotalSize     bool
```

Comment:
This would part of the interface definition, each implementation would return a value on whether they support it or not

Reply:
We don't have a standard interface currently used by all the folder checks?

We can make a function on the Connection though?